cutevariant/gui/mainwindow.py

```diff
@@ -357,6 +359,11 @@ def currentView(self):
             return None
         return self.tab_view.currentWidget()
 
+    @Slot()
```

Comment:
@Slot keywords is not necessary when you connect with : ` sender.connect(self.receiver)`
But It may be useful to inform this is a slot method . So, we can keep it .

Reply:
Ok, i think it will be better in the doc